core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -668,6 +688,35 @@ oppia.directive('conversationSkin', [function() {
           return $scope.windowWidth >= TWO_CARD_THRESHOLD_PX;
         };
 
+        $scope.isNarrowViewport = function() {
+          return $scope.windowWidth < TWO_CARD_THRESHOLD_PX;
+        };
+
+        $scope.isTutorCardDisplayedOnNarrow = function() {
+          if (!$scope.isCurrentSupplementalCardNonempty()) {
+            return $scope.isNarrowViewport();
+          }
+          return $scope.isNarrowViewport() &&
+                 $scope._tutorCardDisplayedOnNarrow;
+        };
+
+        $scope.isSupplementalCardDisplayedOnNarrow = function() {
+          return $scope.isNarrowViewport() &&
+                 !$scope._tutorCardDisplayedOnNarrow;
+        };
+
+        $scope.displayTutorCardOnNarrow = function() {
+          if ($scope.isNarrowViewport()) {
+            $scope._tutorCardDisplayedOnNarrow = true;
+          }
+        };
+
+        $scope.displaySupplementaryCardOnNarrow = function() {
```

Comment:
Note: 'supplemental', not 'supplementary'.

Also suggest 'on' --> 'if'. 'on' suggests an event handler.